### PR TITLE
[MBL-17267][Student][Teacher] - Implement Calendar Screen E2E tests

### PR DESCRIPTION
**Pull request by kdeakinstructure**
Implement Calendar Screen E2E test in both the Student and Teacher apps
Add page object methods.

refs: MBL-17267
affects: Teacher, Student
release note: none

### CODE REVIEW

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Parent Install Page</p>

File: apps/student/src/androidTest/java/com/instructure/student/ui/e2e/compose/CalendarE2ETest.kt [hunk 1]
```diff
@@ -203,4 +204,115 @@ class CalendarE2ETest: StudentComposeTest() {
...
+        Log.d(STEP_TAG, "Login with user: '${student.name}', login id: '${student.loginId}'.")
+        tokenLogin(student)
+        dashboardPage.waitForRender()
+
+        Log.d(STEP_TAG, "Click on the 'Calendar' bottom menu to navigate to the Calendar page. Assert that the page title is 'Calendar' and there is no item in it yet.")
+        dashboardPage.clickCalendarTab()
+        calendarScreenPage.assertCalendarPageTitle()
+        calendarScreenPage.assertEmptyView()
+
+        //Calendar expand/collapse state remains sometimes from previous runs and this way we are handling whether if it's starting from a collapsed or expanded state.
```
Comment (hermannakos) [inline apps/student/src/androidTest/java/com/instructure/student/ui/e2e/compose/CalendarE2ETest.kt:227]: We should clear the prefs before the test runs, so we don't have this problem.

**Review by hermannakos — CHANGES_REQUESTED** [review]

**Review by tamaskozmer — APPROVED** [review]

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Parent Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Review by hermannakos — APPROVED** [review]

**Pull request merged**